spec/takeaway_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu) }
+  let(:menu) { double :menu }
+  
+
+  describe '#show_menu' do
+
+    it 'returns a hash' do
+      allow(menu).to receive(:show).and_return(Hash.new)
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)